docs/explanation/upgrade.rst

```diff
@@ -27,6 +27,12 @@ in a sequential manner, performing upgrades on units within a single zone in par
 *all-in-one* method for upgrades. because its charm is designed to maintain service
 availability throughout the upgrade process.
 
+
+SRULP support
+~~~~~~~~~~~~~
+
+COU follows Charmed OpenStack's upgrade strategy. So SRULP is not supported by COU.
```

Comment:
Acronym needs fixing here too. Also the two sentences may need to be tied together more - it's not clear that charmed openstack does not support SLURP here.